app.js

```diff
@@ -1,25 +1,68 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express");
+const mongoose = require("mongoose");
+const cors = require("cors");
+const contactsRouter = require("./routes/api/contacts");
+const authRouter = require("./routes/api/authRouter");
+const authController = require("./controllers/authController");
+const authMiddleware = require("./middleware/authMiddleware");
+require("dotenv").config({ path: "./mongo.env" });
 
-const contactsRouter = require('./routes/api/contacts')
+const nodemailer = require("nodemailer");
 
-const app = express()
+const transport = nodemailer.createTransport({
+  host: "smtp.meta.ua",
```

Comment:
you could separate nodemailer config into another file. No need for it to be in app.js